Update Virtual Network Description

cameronmotenMSFT opened the pull request:
Confusing wording on that you cannot "remove" a nic on an Azure vm with the portal. However you can "detach" and then delete a NIC as I have tested today on the ms portal.

azurecla commented:
Hi __@singhkay__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (kasing). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

cameronmotenMSFT committed:
Update Virtual Network Description

Confusing wording on that you cannot "remove" a nic on an Azure vm with the portal. However you can "detach" and then delete a NIC as I have tested today on the ms portal.
